src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -38,6 +39,16 @@ object SlickDataAccess {
 
   implicit class ConfigWithUniqueSchema(val config: Config) extends AnyVal {
     /**
+     * Returns either the "url" or "properties.url"
+     */
+    def urlKey = if (config.hasPath("url")) "url" else "properties.url"
+
+    /**
+     * Returns the value of either the "url" or "properties.url"
+     */
+    def urlValue = config.getString(urlKey)
```

Comment:
When using HikariCP, the default Slick 3.0.0 connection pool, with the newer `javax.sql.DataSource`, the `urlKey` is  `properties.url`. For other combinations, the key is `url`.

> The following additional keys are supported for HikariCP only:
> • dataSourceClass (String, optional): ... Note that url is ignored when this key is set (You have to use properties to configure the database connection instead).
> …

-- [Database.forConfig](http://slick.typesafe.com/doc/3.0.0/api/index.html#slick.jdbc.JdbcBackend%24DatabaseFactoryDef@forConfig(String,Config,Driver):Database)